packages/react/src/components/Datepicker/index.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import DayPickerInput from 'react-day-picker/DayPickerInput';
+import 'react-day-picker/lib/style.css';
+import Icon from '../Icon';
+import IconButton from '../IconButton';
+
+const DatepickerNavigation = React.forwardRef(
+  ({ month, onPreviousClick, onNextClick }, ref) => {
+    return (
+      <div ref={ref} className='f-Datepicker--navbar'>
+        {month.toLocaleString('default', { month: 'long' })}{' '}
+        {month.getFullYear()}
+        <div className='f-Datepicker--actions'>
+          <IconButton
+            onClick={() => onPreviousClick()}
+            icon={Icon.ICONS.IconChevronLeft}
+          />
+          <IconButton
+            onClick={() => onNextClick()}
+            icon={Icon.ICONS.IconChevronRight}
+          />
+        </div>
+      </div>
+    );
+  },
+);
+
+const DatepickerInput = React.forwardRef(
+  ({ onClick, placeholder, value, isInvalid }, ref) => (
+    <div
+      ref={ref}
+      onClick={onClick}
+      className={'f-FormEl f-FormEl--withIcon f-Datepicker'}
+    >
+      {value || placeholder}
+      <Icon
+        icon={isInvalid ? Icon.ICONS.IconAlertOctagon : Icon.ICONS.IconCalendar}
+        className='f-FormEl-icon'
+      />
+    </div>
+  ),
+);
+
+const Datepicker = ({ className, isInvalid, placeholder, value, onChange }) => {
+  return (
+    <div
+      className={cx('f-FormEl-wrapper f-Datepicker', {
```

Comment:
Why not using a Dropdown here?
With it, we would benefit from the "clicking outside will hide it" feature and improve the mobile display

Reply:
I just used the in-built hide/show of react-day-picker. I can use the dropdown instead :+1:

Reply:
Fixed